kebechet/managers/thoth_provenance/README.rst

```diff
@@ -0,0 +1,86 @@
+Kebechet Thamos-Provenance_check Manager
+-----------------------
+
+The purpose of this manager is to use Thoth's ability to conduct provenance checks to ensure that your Python dependency sources are valid
+file, it uses `thamos.lib` to communicate with `thoth` user API
+
+A prerequisite for this manager is to have `Pipfile` and `Pipfile.lock`, and `.thoth.yaml`, present in the repo.
+`Pipfile` should state all direct dependencies (with possible required specifications).
+`Pipfile.lock` should hold all the lock data associated with the Python software stack generated either by `thamos.lib.advise()` or `pipenv sync`
+`.thoth.yaml` should contain a valid configuration for `thamos`
+
+* `Pipfile` - respecting `pipenv <https://github.com/pypa/pipenv>`_ tool
+* `Pipfile.lock` - states all pinned down versions of your application stack
+
+Custom PyPI indexes are supported respecting `Pipfile` syntax.
```

Comment:
Please link to https://pipenv.readthedocs.io/en/latest/advanced/#specifying-package-indexes